.github/.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "console": "integratedTerminal",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            //"preLaunchTask": "npm: distribute - actions/build_cache_executor",
+            "program": "${file}",
+            "outFiles": [
+                "${workspaceFolder}/dist/**/*.js"
+            ]
+        },
```

Comment:
I was just a shortcut that would build before starting debugging (if preLaunchTask is uncommented). Otherwise, _in the current state_ it does the default "node debug" behaviour of debugging the current file. More custom things can be done if, let's say you want to spend the time for making the debugger debug on the ts instead of the generate js etc. So it has some usefulness mainly if more time is spent on it, but it can be removed if you want.